src/containers/shared/components/Currency.tsx

```diff
@@ -10,14 +13,17 @@ interface Props {
 
 const Currency = (props: Props) => {
   const { issuer, currency, link } = props;
+  const currencyCode =
+    currency?.length === NON_STANDARD_CODE_LENGTH ? hexToString(currency) : currency;
+
   const content = issuer ? (
     <>
-      {currency}
+      {currencyCode}
       .
       <Account account={issuer} link={link} />
     </>
```

Comment:
I'm surprised you can use empty tags `</>` in React. What do they get rendered as by default? Just curious 😀

Reply:
They look to be rendered as nothing.  Seems to be a way to get around having a wrapping element.